packages/skeleton-react/src/lib/components/Accordion/Accordion.tsx

```diff
@@ -1,39 +1,29 @@
 'use client';
 
-import { motion, AnimatePresence } from 'framer-motion';
-import React, { createContext, useContext, useEffect, useState } from 'react';
+import { createContext, useContext, useId } from 'react';
+import type { FC } from 'react';
+import * as accordion from '@zag-js/accordion';
+import { useMachine, normalizeProps } from '@zag-js/react';
 
-import {
-	AccordionContextState,
-	AccordionControlProps,
-	AccordionItemContextState,
-	AccordionItemProps,
-	AccordionPanelProps,
-	AccordionProps
-} from './types.js';
+import type { AccordionContextState, AccordionControlProps, AccordionItemProps, AccordionPanelProps, AccordionProps } from './types.js';
+import { noop } from '../../internal/noop.js';
 
 // Contexts ---
 
 export const AccordionContext = createContext<AccordionContextState>({
 	animDuration: 0.2,
```

Comment:
AccordionContextState has a comment that `animDuration` should be in milliseconds. The Svelte implementation passes `200`.

Reply:
The default value supplied is correct, it's the JSdoc comment that's wrong. In Svelte we use the Svelte transitions system which takes values as milliseconds (ex: `200`), but in React we're leaning into Zag to control animations, which takes them as float values (ex: `0.2`). So I've updated the React JSdoc comment to make this more clear.